api/envoy/extensions/health_checkers/thrift/v3/thrift.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package envoy.extensions.health_checkers.thrift.v3;
+
+import "envoy/extensions/filters/network/thrift_proxy/v3/thrift_proxy.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.health_checkers.thrift.v3";
+option java_outer_classname = "ThriftProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/health_checkers/thrift/v3;thriftv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Thrift]
+// Thrift health checker :ref:`configuration overview <config_health_checkers_thrift>`.
+// [#extension: envoy.health_checkers.thrift]
+
+// [#next-free-field: 6]
+message Thrift {
+  reserved 1, 5;
+
+  reserved "service_name", "request_headers_to_add";
+
+  // Specifies the method name will be set to each health check request to an upstream host.
```

Comment:
Specifies the method name that will be set on each health check request dispatched to an upstream host.